tcms/rpc/tests/test_testcase.py

```diff
@@ -16,29 +21,96 @@
                                   TestPlanFactory, UserFactory, VersionFactory)
 
 
-class TestNotificationRemoveCC(APITestCase):
-    """ Tests the XML-RPC testcase.notication_remove_cc method """
+class TestValidateEmail(unittest.TestCase):
+    def test_non_list_email_collection(self):
+        with self.assertRaisesRegex(TypeError, 'cc_list should be a list object.'):
+            _validate_cc_list('example@example.com')
+
+    def test_invalid_email(self):
+        with self.assertRaises(ValidationError):
+            _validate_cc_list(['@example.com'])
+
+    def test_valid_email_list(self):  # pylint: disable=R0201
```

Comment:
please use the human-readable message ID when disabling pylint errors. was that `no-self-use` ?